tests/kernel/shutdown_ltp.pm

```diff
@@ -14,8 +14,9 @@ use 5.018;
 use warnings;
 use base 'opensusebasetest';
 use testapi;
-use utils;
 use power_action_utils 'power_action';
+use utils;
```

Comment:
Is there a reason you changed the order?

Reply:
Alphabetic order, but I'll revert it.

Reply:
OK, I just like to have git-blame work the best way possible and also if you change the order it makes me think you had some load order problem. Thanks.